packages/augur-tools/src/templates-source.ts

```diff
@@ -9772,6 +9773,39 @@ export const TEMPLATES = {
           },
         ],
       },
+      [DEFI]: {
+        templates: [
+          {
+            marketType: SCALAR,
+            question: `What will the Total Value Locked "USD" in Defi be on [0] (UTC), according to https://defipulse.com/ ?`,
+            example: `What will the Total Value Locked "USD" in Defi be on 03/27/2022 (UTC), according to https://defipulse.com/ ?`,
+            denomination: 'Price',
+            inputs: [
+             {
+                id: 0,
+                type: TemplateInputType.DATEYEAR,
+                daysAfterDateStart: 1,
+                placeholder: `Day of Year`,
+              },
+            ],
+            resolutionRules: {
+              [REQUIRED]: [
+                {
+                  text: `This market shall resolve based on the Total Value Locked (USD) in DeFi reported by defipulse.com at the specified time. If data is unavailable at defipulse.com, the market shall resolve according to the DeFi pulse API.`,
+                  text: `To report, choose "30D" in the Total Value Locked (USD) for ALL DeFi. Hover over points until you reach the specified date and report the TVL (USD) in Billions (B) with 3 decimal points. If more are available, truncate the third decimal point following rounding rules: if the fourth decimal is 5 or greater, round up. Otherwise remain the same.`,
+                },
+                {
+                  text: `If using the DefiPulse public API found at https://public.defipulse.com/api/GetHistory?period=1m&api-key=<API_KEY> and report the value shown for the required date.`,
```

Comment:
Are API keys free?  If you have to pay or KYC to get them then I don't think this can reasonably used as a reporting source.  May be worth mentioning how to get an API_KEY (e.g., a helpful but non-authoritative link).

Reply:
This API call looks like it only gets recent history.  For reporting we need a source that can be accessed (with reliable results) many months after reporting starts.

Reply:
So the API allows up to a year and probably could be more. I imagine within a year they'll beef this up too. I added some information about what could possibly be called as long as it's grouped by date, and included a link to the documentation.

Reply:
Also, API keys are free, but I've also removed that. This endpoint is public so you don't actually need an API key.